neutrons/histogram-dev
# Pkg conda python

# Short description of the changes:
The code includes changes for creating python wheel and conda package
lib/ and bin/ directories are not included. Only python code in src/ directory.

# Long description of the changes:
It includes:
- folders moving and renaming, remove extra files
- pyproject.toml, conda.recipe 
- environment.yml updates to include only minimal dependencies
- README for the building process

# Check list for the pull request

# Check list for the reviewer

# Manual test for the reviewer
Package Build and Installation Instructions

Python wheel

   python -m build --wheel --no-isolation
   check-wheel-contents dist/histogram_*.whl

Conda package

  // create a conda package
   cd conda.recipe
   echo "versioningit $(versioningit ../)"
   CHANNELS="--channel mcvine --channel conda-forge"
   VERSION=$(versioningit ../) conda mambabuild $CHANNELS --output-folder . .
   conda verify noarch/histogram-*.tar.bz2
  // install a local conda package
   conda install noarch/<histogram .tar.bz2 file>

# References
EWM: [6590 - Package Installation](https://ornlrse.clm.ibmcloud.com/ccm/resource/itemName/com.ibm.team.workitem.WorkItem/6590)
